### PR TITLE
nullarbor 2.0.20191003

### PR DESCRIPTION
**Pull request by tseemann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/nullarbor/meta.yaml [hunk 1]
```diff
@@ -1,70 +1,67 @@
...
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
 
 requirements:
   run:
-    - perl
```
Comment (bgruening) [inline recipes/nullarbor/meta.yaml]: please leave perl in

**Comment by dfornika** [comment]
@tseemann maybe use this to create a doi for the repo?

https://guides.github.com/activities/citable-code/

File: recipes/nullarbor/meta.yaml (hunk 1 above)
Comment (tseemann) [reply recipes/nullarbor/meta.yaml]: ok, i assumed the perl mod deps would take care of that. but i will add back.

**Comment by tseemann** [comment]
@bgruening no DOI yet. I considered Zenodo etc, but I am unable to use those DOIs in my grant applications because they are not technically peer reviewed :-(

File: recipes/nullarbor/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/nullarbor/meta.yaml]: This is technically true. But we use those explicit language packages to determine against how many different versions we need to build the package. This is more important for python/ruby/R, but hopefully someone will care about the perl ecosystem at some point and then its good to have this in place - to be consistent.

**Pull request merged**